src/System.Management.Automation/engine/remoting/commands/EnterPSHostProcessCommand.cs

```diff
@@ -91,6 +92,17 @@ public PSHostProcessInfo HostProcessInfo
             set;
         }
 
+        /// <summary>
+        /// Gets and sets the Named Pipe name to connect to.
+        /// </summary>
+        [Parameter(Position = 0, Mandatory = true, ValueFromPipeline = true, ParameterSetName = EnterPSHostProcessCommand.PipeNameParameterSet)]
+        [ValidateNotNull()]
```

Comment:
`ValidateNotNull` is not needed. `Mandatory = true` implies no-null and no-empty input.

Reply:
removed.